EIPS/eip-5851.md

```diff
@@ -0,0 +1,255 @@
+     * example: 
+    {   "title": "jurisdiction",
+        "_type": "string",
+        "description": "two-word code defining legal jurisdiction"
+        }
+    * @notice it can be further optimized by using efficient encoding schemes (like TLV etc) and there can be a tradeoff in the gas costs of storing large strings vs encoding/decoding costs while describing the standard.
+     */
+    struct Metadata {
+        string title;
+        string _type;
+        string description;
+    }
+    /**
+     * @dev Values here can be read and written by smart contract and front-end, cited from [EIP-3475].
+     example : 
+{
+ jurisdiction = IERC5851.stringValue("CH");
+}
+     */
+    struct Values { 
+        string stringValue;
+        uint uintValue;
+        address addressValue;
+        bool boolValue;
+    }
+```
+
+**2. Requirement structure:**
+This will be stored in each of the SBT certificates that will define the conditions that need to be satisfied by the owner of the certificate when any verifier (of the DeFI protocol functions) calling the `isVerified()` function, to be validated to run the function, this will be defined for each onchain `Values` separately.
+
+```solidity
+    /**
+     * @dev structure that DeFines the parameters for specific requirements of the SBT certificate
+     * @notice this structure is used for the verification process, it contains the metadata, logic, and expectation
+     * @logic given here MUST be one of ("⊄", "⊂", "<", "<=", "==", "!=", ">=",">")
+     ex: standardRequirement => {
+    { "title": "adult",
+        "type": "uint",
+        "description": "client holders age to be gt 18 yrs.",
+        },
+       "logic": ">=",
+    "value": "18" 
+}
+Defines the condition encoded for the identity index 1, DeFining the identity condition that the holder must be more than 18 years old.
+    */
+    struct Requirement {
+        Metadata metadata;
+        string logic;
+        Values expectation;
+    }
+```
+
+## Rationale
+
+For the standard interface functions, we followed the implementation of the W3C DID specification, which defines the components of the data registry, DID identifier. thus we represented the same properties in our standard using the Verifier and SBT standard. Also to define the metadata (URL, topics, subjects, and the identifier), we followed the metadata definition as [eip-3475](./eip-3475.md) with the possibility to define context metadata and verification condition within the single contract.
+
+## Backwards Compatibility
+
+- The ERC standard remains backward compliant for previous versions for cases that only do the changes in schema of the requirements structure.
+- In case of the changes like the logic of verification protocol, developers will have to use proxy contract patterns like [eip-1967](./eip-1967.md) which will route the `isVerified()`function call based on the version of verifier contracts the SBT was issued.
```

Comment:
```suggestion
- In case of the changes like the logic of verification protocol, developers will have to use proxy contract patterns like [EIP-1967](./eip-1967.md) which will route the `isVerified()` function call based on the version of verifier contracts the SBT was issued.
```
Not sure if this sentence should be in this section. What you have here is forwards compatibility, not backwards compatibility. This is useful nonetheless though.